packages/core/src/node/backend-application.ts

```diff
@@ -152,12 +151,11 @@ export class BackendApplication {
     @inject(ProcessUtils)
     protected readonly processUtils: ProcessUtils;
 
-    private readonly _performanceObserver: PerformanceObserver;
-
     constructor(
         @inject(ContributionProvider) @named(BackendApplicationContribution)
         protected readonly contributionsProvider: ContributionProvider<BackendApplicationContribution>,
-        @inject(BackendApplicationCliContribution) protected readonly cliParams: BackendApplicationCliContribution
+        @inject(BackendApplicationCliContribution) protected readonly cliParams: BackendApplicationCliContribution,
+        @inject(Stopwatch) protected readonly stopwatch: Stopwatch
```

Comment:
This can be added as a property injection.

Reply:
The constructor calls `initialize`, which needs to use the stopwatch to measure the initializations of contributions. But this happens before property injection, so constructor injection seemed to be the way to go. This then would also have to be added to the breaking API changes in the changelog.

I would have moved the `initialize` step to the `@postConstruct configure` method, but I figured there must have been a reason to call it from the constructor and so I wasn't comfortable making that change.